pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[project]
+name = "PatientX"
+version = "0.1.0"
+
+license = { text = "MIT" }
+
+readme = "README.md"
+
+dependencies = [
```

Comment:
Add version info. Consider using `uv` if you don't want to manage the dependencies manually.